internal/tools/aws/constants.go

```diff
@@ -56,6 +56,31 @@ const (
 	// Note: This needs to be manually created before RDS databases can be used.
 	DefaultDBSubnetGroupTagValue = "MYSQL/Aurora"
 
+	// DefaultInstallCertificatesTagKey is the default key used to find the server
+	// TLS certificate ARN.
+	// Note: This needs to be manually created before attempting to provision a cluster in VPC.
```

Comment:
This `Note` should probably be removed. It will apply to all clusters.

I believe this applies to your other constants as well.